### PR TITLE
MMT-2232/2233 Revision viewing and reversion for tools

### PR DESCRIPTION
**Pull request by ryanmiller-1**
Updating a subscriptions ticket that was causing intermittent failures.

### CODE REVIEW

File: spec/features/tools/reverting_tool_spec.rb [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      context 'when visiting the revisions page from a different provider' do
+        before do
+          login(provider: 'MMT_1', providers: %w(MMT_1 MMT_2))
+
+          visit tool_revisions_path(@multiple_revisions_ingest_response['concept-id'])
+        end
+
+        it 'displays all the correct revision information' do
+          within 'main header' do
+            expect(page).to have_content('Reverting Services Test -- revision 04')
```
Comment (william-valencia) [inline spec/features/tools/reverting_tool_spec.rb:60]: Should this be "Reverting Tools Test"?

File: spec/features/tools/reverting_tool_spec.rb [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+          context 'when clicking Yes' do
+            before do
+              find('.not-current-provider-link').click
+              wait_for_jQuery
+            end
+
+            it 'reverts the tool to the correct revision and displays the correct revision information and switches provider context' do
+              within 'main header' do
+                expect(page).to have_content('Reverting Services Test -- revision 01')
```
Comment (william-valencia) [inline spec/features/tools/reverting_tool_spec.rb:92]: Should this be "Reverting Tools Test"?

File: spec/features/tools/reverting_tool_spec.rb [hunk 3]
```diff
@@ -0,0 +1,169 @@
+describe 'Reverting to previous tools', reset_provider: true, js: true do
+  before :all do
+    # tool for simple reverting tool test
+    @simple_revert_ingest_response, @simple_revert_concept_response, @native_id = publish_tool_draft(revision_count: 2)
+
+    # tool for reverting tool with many revisions
+    @multiple_revisions_ingest_response, @multiple_revisions_concept_response, @native_id2 = publish_tool_draft(revision_count: 4, long_name: 'Reverting Services Test', number_revision_long_names: true)
```
Comment (william-valencia) [inline spec/features/tools/reverting_tool_spec.rb:7]: Should be Tools not Services.

File: spec/features/tools/reverting_tool_spec.rb (hunk 1 above)
Comment (ryanmiller-1) [reply spec/features/tools/reverting_tool_spec.rb]: Yes, my mistake.

**Pull request merged**